src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,590 @@
+        self.acctGroup = getattr(config.BossAir, 'acctGroup', "production")
+        self.acctGroupUser = getattr(config.BossAir, 'acctGroupUser', "cmsdataops")
+
+        # Build a requirement string
+        self.reqStr = "stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS=?=\"any\") || (GLIDEIN_REQUIRED_OS=?=REQUIRED_OS)) && (TARGET.Cpus >= RequestCpus)"
+        if hasattr(config.BossAir, 'condorRequirementsString'):
+            self.reqStr = config.BossAir.condorRequirementsString
+
+        # x509 proxy handling
+        proxy = Proxy({'logger': myThread.logger})
+        self.x509userproxy = proxy.getProxyFilename()
+        self.x509userproxysubject = proxy.getSubject()
+
+        return
+
+    def submit(self, jobs, info=None):
+        """
+        _submit_
+
+
+        Submit jobs for one subscription
+        """
+
+        # If we're here, then we have submitter components
+        self.scriptFile = self.config.JobSubmitter.submitScript
+
+        successfulJobs = []
+        failedJobs = []
+
+        if len(jobs) == 0:
+            # Then was have nothing to do
+            return successfulJobs, failedJobs
+
+        schedd = htcondor.Schedd()
+
+        # Submit the jobs
+        for jobsReady in grouper(jobs, self.jobsPerSubmit):
+
+            cluster_ad = self.getClusterAd()
+            proc_ads = self.getProcAds(jobsReady)
+
+            logging.debug("Start: Submitting %d jobs using Condor Python SubmitMany" % len(proc_ads))
+            try:
+
+                clusterId = schedd.submitMany(cluster_ad, proc_ads)
+
+            except Exception as ex:
+
+                logging.error("SimpleCondorPlugin job submission failed.")
+                logging.error("Moving on the the next batch of jobs and/or cycle....")
+                logging.exception(ex)
+
+                condorErrorReport = Report()
+                condorErrorReport.addError("JobSubmit", 61202, "CondorError", str(ex))
+                for job in jobsReady:
+                    job['fwjr'] = condorErrorReport
+                    failedJobs.append(job)
+
+                # Report submission error to operators
+                # TODO - Dirk: Do we really need this ?
+                try:
+                    from WMCore.Alerts import API as alertAPI
+                    preAlert, sender = alertAPI.setUpAlertsMessaging(self,
+                                                                     compName="BossAir.SimpleCondorPlugin")
+                    sendAlert = alertAPI.getSendAlert(sender=sender,
+                                                      preAlert=preAlert)
+                    sendAlert(6, msg="Condor submission failed. Check condor status.")
+                    sender.unregister()
+                except:
+                    pass
+
+            else:
+
+                logging.debug("Finish: Submitting jobs using Condor Python SubmitMany")
+                for index,job in enumerate(jobsReady):
```

Comment:
I am pretty sure, but @bbockelm might want to comment. It's the only way to do this anyways since submitMany doesn't return anything except the clusterId.

The assumption here is that submitMany will submit jobs with procId 0...N in an unbroken sequences in the same order the jobs are passed in the submitMany call.